File: BackEnd/TalkingWithFredsAPI/TalkingWithFredsAPI/Models/ConvMessages.cs
using System;
using System.Collections.Generic;

namespace TalkingWithFredsAPI.Models
{
    public partial class ConvMessages
    {
        public int MessageId { get; set; }
        public int SenderId { get; set; }
        public int ConvId { get; set; }
        public DateTime MessageSendingTime { get; set; }
        public string MessageContent { get; set; }

        public Conversations Conv { get; set; }
        public Users Sender { get; set; }
    }
}

File: FrontEnd/talking-with-freds/src/common/models/ConvMessages.ts
import Conversation from "./Conversation";
import User from "./User";

export default class ConvMessage {
  messageId: number = 0;
  senderId: number = 0;
  convId: number = 0;
  messageSendingTime: Date = new Date();
  messageContent: string = '';
  conv: Conversation = new Conversation();
  sender: User = new User();
}
